bundles/binding/org.openhab.binding.zwave/database/aeon/zw096.xml

```diff
@@ -191,7 +191,6 @@
             <Type>byte</Type>
             <Default>DDA0DD</Default>
             <Minimum>0</Minimum>
-            <Maximum>FFFFFF</Maximum>
```

Comment:
We probably shouldn't remove this - however it should be in decimal rather than hex